.travis.yml

```diff
@@ -105,6 +105,8 @@ matrix:
           services:
               - docker
           before_install:
+              - nvm install
+              - nvm use
```

Comment:
Right, I think this needs a `.nvmrc` - of course I would prefer to do this without another file. Let's see if it works like that and otherwise, switch to specifying the version on the commandline.